season-config.html

```diff
@@ -15,14 +15,16 @@ <h3 id="newSeasonConfirmation"></h3><br><br>
 <br><br><br><br>
 <h3 id = "seasonQuestionsTitle" style = "text-align: center"></h3>
 
-<h3>Enter in any scoring opportunities in the first text box and how many points that scoring opportunity is worth in the second text box.
-    <br><br>
-    Also, please enter the type of answer to be expected when answering questions about the scoring opportunity in 
-    the Match Data section of <a href='/event-data.html'>Team Data</a>.<br>
-    For example, if the scoring opportunity was 'ball,' the input type would probably be numeric. <br>
-    However, if the scoring opportunity was 'endgame climb,' the input type would probably be text ('yes the team can', 'no the team can't').
+<h3>Enter in any things to record actively when watching matches in 
+    the Match Data section of <a href='/event-data.html'>Team Data</a>. <br><br>
+    ● For scoring opportunities like elements and end game actions, enter in their name in the first box and how many points the
+    action is worth in the second box.<br>
+    ● For the input type, enter in 'Numeric' (Users can enter in 0 for things like "no endgame climb").<br><br>
+    ● If you would like to also enter in questions here to record specific in-game characteristics of a team 
```

Comment:
For scoring opportunities like elements and end game actions, enter in their name in the first box and how many points the action is worth in the second box.

For the input type, choose 'Numeric' to allow users to enter a number value and 'Text' to allow users to enter arbitrary text.